src/Compatibility/ControlGallery/src/Directory.Build.targets

```diff
@@ -0,0 +1,4 @@
+<Project>
+  <Import Project="$(MauiSrcDirectory)Maui.InTree.targets" Condition=" '$(UseMaui)' != 'true' " />
+  <Import Project="$(MauiRootDirectory)Directory.Build.targets" />
```

Comment:
Add the `Maui.InTree.targets` riiiight at the end of the list, but before the parent `Directory.Build.targets`. The only way to do this is to include the `Maui.InTree.targets` in a local `Directory.Build.targets`, and then include the parent afterwards.